src/components/accountForm/SignIn/SignIn.jsx

```diff
@@ -52,7 +52,7 @@ const SignIn = ({ header, newUserHandler, onFormSubmit, onLogIn }) => {
         }
 
         if (dataUsers.filter((user) => user.email === email).length === 0) {
-            newErrors.email = 'E-mail is not in a database';
```

Comment:
the problem is that before we check if  email is in database we should check if email format is valid. this is the case. If email is valid then we should check if user is in database. neverless we will send request to check it